diff --git a/config.def.h b/config.h
index b10675c..d8629a5 100644
--- a/config.def.h
+++ b/config.h
@@ -2,8 +2,6 @@
 
 /* appearance */
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
-static const int startwithgaps	     = 0;	 /* 1 means gaps are used by default */
-static const unsigned int gappx     = 10;       /* default gap between windows in pixels */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
@@ -104,7 +102,7 @@ static Key keys[] = {
 	TAGKEYS(                        XK_8,                      7)
 	TAGKEYS(                        XK_9,                      8)
 	{ MODKEY|ShiftMask,     XK_r,      reload,           {.i=23} },
-	{ MODKEY|ShiftMask,		XK_q,	  spawn,	  {.v = quit} },
+	{ MODKEY|ShiftMask,		XK_q,	  spawn,	  {.v = quit}},
 	{ MODKEY|ShiftMask,		XK_f,					spawn,			{.v= firefox} },
 };
 
